dist/Panda.json

```diff
@@ -592,9 +605,9 @@
     "extensionButton.prominentBackground": "#45A9F9",
     "extensionButton.prominentHoverBackground": "#676B79",
     "editorError.foreground": "#FF4B82",
-    "editorError.border": "#FF4B82",
+    "editorError.border": "#292A2B",
     "editorWarning.foreground": "#FFCC95",
-    "editorWarning.border": "#FFCC95",
+    "editorWarning.border": "#292A2B",
```

Comment:
This one is from some previous commit. Seems like Panda.json had not been updated.